exercises/K-functions-parameters/exercise3.js

```diff
@@ -1,4 +1,6 @@
-// Write your function here
+function createGreeting(hello,my,name,is,Daniel){ 
```

Comment:
This is incorrect. The purpose of parameters is to be able to dynamically manipulate data. In this instance you would want a name to be dynamically added to a greeting message. You'd therefore just need the one parameter which would be the name. You've also 'hardcoded' (non dynamically programmed) the greeting message so this in effect makes all of your function parameters unnecessary. It should look somehting like this:

function createGreeting(name) {
   return "Hello my name is " + name;  
}

You can see the parameter is passed to the return string so that the message can be dynamic.